arangod/Aql/QueryCursor.h

```diff
@@ -24,9 +24,11 @@
 
 #pragma once
 
+#include "Aql/Query.h"
 #include "Aql/QueryResult.h"
 #include "Aql/SharedAqlItemBlockPtr.h"
 #include "Basics/Common.h"
+#include "StorageEngine/TransactionState.h"
```

Comment:
The 2 new includes don't seem to be needed by the new code added to this header. Can we move them to the .cpp file?